src/aura/RD2_EnablementApp/RD2_EnablementApp.app

```diff
@@ -38,4 +38,5 @@
     <aura:dependency resource="c:RD2_EnablementDelegateSimple" />
     <aura:dependency resource="c:rd2StatusMappingSettings" />
     <aura:dependency resource="c:utilIllustration" />
+    <aura:dependency resource="c:bdeRetirementAlert" />
```

Comment:
I put in here because I was trying not to create a lightning out app just for this retirement alert, I'm fine to create a lightning app just for this but we'll be making it redundant soon. What are your thoughts?

Reply:
Let's not create separate lightning app just for this purpose if we can avoid it